scripts/HGDP_SampleSelection.ipynb

```diff
@@ -0,0 +1,1716 @@
+{
```

Comment:
<p>By the way, I don't think any Papuan samples survive this filtration, which is unfortunate.. If we go ahead with downloading Papuan CRAMs, they'll end up getting filtered out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>